acceptance/gossip_peerings_test.go

```diff
@@ -140,9 +140,9 @@ func TestGossipRestart(t *testing.T) {
 	log.Infof("waiting for range replication")
 	checkRangeReplication(t, l, 10*time.Second)
 
-	log.Infof("stopping all nodes")
+	log.Infof("killing all nodes")
```

Comment:
potentially faster. In the unlikely event that we're killing the node while an internal client is doing something under a stopper task (which needs a majority), it would take five seconds.